README.md

```diff
@@ -20,95 +20,63 @@ More information about releases in Octopus Deploy:
 Incorporate the following actions in your workflow to create a release in Octopus Deploy using an API key, a target instance (i.e. `server`), and a project:
 
 ```yml
+env:
+  
+
 steps:
   # ...
   - name: Install Octopus CLI 🐙
     uses: OctopusDeploy/install-octopus-cli-action@v1
     with:
       version: latest
+
   - name: Create a release in Octopus Deploy 🐙
-    uses: OctopusDeploy/create-release-action@v1
+    uses: OctopusDeploy/create-release-action@v2
+    env:
+      OCTOPUS_API_KEY: ${{ secrets.API_KEY  }}
+      OCTOPUS_HOST: ${{ secrets.SERVER }}
+      OCTOPUS_SPACE: 'Spaces-1'
     with:
-      api_key: ${{ secrets.API_KEY }}
-      project: 'Projects-123'
-      server: ${{ secrets.SERVER }}
+      project: 'MyProject'
 ```
 
-Here's an example that provides a `username` and `password` to authenticate to Octopus Deploy:
+## 📥 Environment Variables
 
-```yml
-steps:
-  # ...
-  - name: Install Octopus CLI 🐙
-    uses: OctopusDeploy/install-octopus-cli-action@v1
-    with:
-      version: latest
-  - name: Create a release in Octopus Deploy 🐙
-    uses: OctopusDeploy/create-release-action@v1
-    with:
-      password: ${{ secrets.PASSWORD }}
-      project: 'Projects-123'
-      server: ${{ secrets.SERVER }}
-      username: ${{ secrets.USERNAME }}
-```
+| Name | Description |
+| :--- | :---------- |
+| `OCTOPUS_API_KEY` | The API key used to access Octopus Deploy. `API-GUEST` may be used if the guest account is enabled. It is strongly recommended that this value retrieved from a GitHub secret. |
+| `OCTOPUS_HOST` | The base URL hosting Octopus Deploy (i.e. `https://octopus.example.com`). It is strongly recommended that this value retrieved from a GitHub secret. |
+| `OCTOPUS_PROXY` | The URL of a proxy to use (i.e. `https://proxy.example.com`). If `OCTOPUS_PROXY_USERNAME` and `OCTOPUS_PROXY_PASSWORD` are omitted, the default credentials are used. It is strongly recommended that this value retrieved from a GitHub secret. |
+| `OCTOPUS_PROXY_PASSWORD` | The password used to connect to a proxy. It is strongly recommended to retrieve this value from a GitHub secret. |
+| `OCTOPUS_PROXY_USERNAME` | The username used to connect to a proxy. It is strongly recommended to retrieve this value from a GitHub secret.|
+| `OCTOPUS_SPACE` | The ID of a space within which this command will be executed. If omitted, the default space will be used. |
```

Comment:
We are moving towards making this a required value across all integrations. Let's remove the text, "If omitted, the default space will be used."